packages/core-modules/contracts/modules/AnotherModuleMock.sol

```diff
@@ -0,0 +1,29 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../mixins/CoreCommsMixin.sol";
+import "./SomeModuleMock.sol";
+
+contract AnotherModuleMock is CoreCommsMixin {
+    /*
```

Comment:
Maybe we can move this giga comment out to documentation and clean up the code a bit?